### ISSUE TITLE
Node 6.0.0 build

### ISSUE THREAD

**Issue by danopia** [pagarme/node-rsa-keygen#2]
Looks like there's some V8 changes in 6.0.0 (from 5.8.0):

```
> rsa-keygen@1.0.5 install /tmp/stuff/node_modules/rsa-keygen
> node-gyp rebuild

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  CXX(target) Release/obj.target/rsa_keygen/src/rsa_keygen.o
In file included from ../src/rsa_keygen.cc:2:
../node_modules/nan/nan.h:590:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:596:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback) {
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:601:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:607:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback) {
      ~~~~~~~~~~~~~^
4 errors generated.
make: *** [Release/obj.target/rsa_keygen/src/rsa_keygen.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/Daniel/.nvm/versions/node/v6.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "/Users/Daniel/.nvm/versions/node/v6.0.0/bin/node" "/Users/Daniel/.nvm/versions/node/v6.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /tmp/stuff/node_modules/rsa-keygen
gyp ERR! node -v v6.0.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok 
```

**Comment by danopia** [comment]
Fixed by #3 , pending a release

**Comment by greenboxal** [comment]
@danopia Sorry, forgot about that. Just released 1.0.6.

**Comment by danopia** [comment]
Thank you for taking the time to fix up this compile problem. All my stuff is green now. Cheers!
